### PR TITLE
added payment method

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference

[Fetch Payment Methods](https://razorpay.com/docs/third-party-validation/s2s-integration/methods-api/)

### CODE REVIEW

File: documents/payment.md [hunk 1]
```diff
@@ -766,6 +766,18 @@ please refer this [doc](https://razorpay.com/docs/payments/third-party-validatio
   "customer_name": "Gaurav Kumar"
 }
 ```
+-------------------------------------------------------------------------------------------------------
+### Fetch payment methods (Third party validation)
+
+```C#
+RazorpayClient client  = new RazorpayClient("key",""); // Use only razorpay key
```
Comment (shwatang) [inline documents/payment.md:773]: Have you tested if this works? I hope passing secret as null is not triggering the Oauth flow.

File: net40/RazorpayClient40.csproj [hunk 2]
```diff
@@ -151,7 +151,10 @@
     </Compile>  
     <Compile Include="..\src\Webhook.cs">
       <Link>Webhook.cs</Link>
-    </Compile>    
+    </Compile>
+	<Compile Include="..\src\Method.cs">
+	  <Link>Method.cs</Link>
+	</Compile>
```
Comment (shwatang) [inline net40/RazorpayClient40.csproj:157]: nit: Indentation fix.

File: documents/payment.md (hunk 1 above)
Comment (ankitdas13) [reply documents/payment.md:773]: Yes, I tested this and its working

Screenshot 2024-05-06 132007

File: net40/RazorpayClient40.csproj (hunk 2 above)
Comment (ankitdas13) [reply net40/RazorpayClient40.csproj]: fixed

**Pull request merged**